modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/utils/constants/getProductName.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {getPascalCase} from '../../../../utils/getPascalCase';
+
+export function getProductName(activationKey) {
+	const productName = getPascalCase(activationKey.licenseEntryType).replace(
+		'-',
+		' '
+	);
+
+	return productName;
+}
```

Comment:
If we are using just the licenseEntryType, should be good pass only this property

```suggestion
export function getProductName(licenseEntryType) {
	return getPascalCase(licenseEntryType)?.replace(
		'-',
		' '
	);
}
```